usr/share/lib/ipa/tests/SLES/Azure/test_sles_azure_accel_networking.py

```diff
@@ -0,0 +1,5 @@
+def test_sles_azure_accel_networking(host):
+    has_mellanox_hw = 'Mellanox' in host.run('sudo lspci').stdout
+
+    if has_mellanox_hw:
+        assert 'mlx4_' in host.run('sudo lsmod').stdout
```

Comment:
Cannot use 'mlx4_' Azure will soon roll out HW that will load the 'mlx5_' driver. We either need to test generically or for either or.